### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by big-r81**
## Checklist

- [ ] Code is written and works correctly
- [ ] Changes are covered by tests
- [ ] Any new configurable parameters are documented in `rel/overlay/etc/default.ini`
- [ ] A PR for documentation changes has been made in https://github.com/apache/couchdb-documentation

### CODE REVIEW

File: FDB_NOTES.md [hunk 1]
```diff
@@ -25,25 +24,25 @@ Things of Note
    this situation we don't run the prep_and_validate code on pre-fdb
    CouchDB. The new code always checks stubs before merging revision trees.
    I'm sure the old way would fail somehow, but it would fail further on
-   which means we may have failed with a different reason (conflict, etc)
+   which means we may have failed with a different reason (conflict, etc.)
    before we got to the next place we check for missing stubs.
 
 6. For multi-doc updates we'll need to investigate user versions on
-   versionstamps within a transaction. Also this likely prevents the
+   version stamps within a transaction. Also this likely prevents the
```
Comment (nickva) [inline FDB_NOTES.md:31]: Grammatically this is correct but in [FDB docs and API](https://apple.github.io/foundationdb/api-python.html?highlight=versionstamp#fdb.tuple.Versionstamp) they are usually referred to as a `versionstamp` all concatenated.

**Review by nickva — CHANGES_REQUESTED** [review]
Thanks for your contribution @big-r81!


+1 after a minor correction

File: FDB_NOTES.md (hunk 1 above)
Comment (big-r81) [reply FDB_NOTES.md]: HI, that makes sense, reverting...

Comment (nickva) [reply FDB_NOTES.md]: Looks great, thank you, for your contribution!

Comment (nickva) [reply FDB_NOTES.md]: Looks great, thank you, for your contribution!

**Review by nickva — APPROVED** [review]

**Pull request merged**